src/shared/components/AddResourceDropdown.tsx

```diff
@@ -47,7 +47,7 @@ class AddResourceDropdown extends PureComponent<Props> {
     const {titleText, status} = this.props
     return (
       <Dropdown
-        style={{width: '232px'}}
+        style={{width: this.props.resourceName === 'Task' ? '180px' : '232px'}}
```

Comment:
Is it possible to make this fit the content instead of forcing a width? I know you just adjusted the previously hardcoded to accomodate tasks, maybe we can just fix this (?)

If not its not a big deal 👍

Reply:
nope, you're absolutely right. I'll look into making the width dynamic